packages/virgo/src/components/virgo-line.ts

```diff
@@ -25,13 +25,12 @@ export class VirgoLine<
     return result;
   }
 
+  protected firstUpdated(): void {
+    this.style.display = 'block';
```

Comment:
Can we setting it on `static style`?

Reply:
I don't know how to construct a css in static css to handle `v-line`.